gcloud/storage/blob.py

```diff
@@ -287,6 +287,9 @@ def download_to_file(self, file_obj, client=None):
         :raises: :class:`gcloud.exceptions.NotFound`
         """
         client = self._require_client(client)
+        if self.media_link is None:  # not yet loaded
+            self.reload()
```

Comment:
Yes, I was implying that we should document the extra API call.

Shouldn't we notify them if the method is going to use 2 HTTP requests instead of 1? That seems to trump the implementation detail.